lib/panel/index.js

```diff
@@ -16,6 +16,7 @@ class Panel {
   hidePanelWhenEmpty: boolean
   showPanelStateMessages: boolean
   activationTimer: number
+  initialActiveBottomPaneItem: PanelDock | null
```

Comment:
Is there a reason you are storing a reference to the `PanelDock`, instead of just a boolean?

Reply:
Not really, just didn't bother converting it. I can make it a boolean if you'd like.